app/api/tenants/tenantContext.ts

```diff
@@ -44,7 +42,7 @@ class Tenants {
 
   async run(cb: () => Promise<void>, tenantName?: string): Promise<void> {
     return new Promise((resolve, reject) => {
-      this.storage.run(tenantName || this.defaultTenantName, () => {
+      this.storage.run(tenantName || config.defaultTenant.name, () => {
```

Comment:
Wondering if this should allow tenantName to be empty and use the conditional in the run, or if it just makes sense to default the function argument to the config value if undefined.  Perhaps not a lot of difference?

Reply:
done 👍